### DISCUSSION TITLE
Upsample/upscale low res image to high res

### DISCUSSION THREAD

**Discussion by dmiyamasu** [dlemstra/Magick.NET#1097] [Help]
I received lots of images via MMS. Average size is < 1.5MB and not very good quality for printing. 

I was wondering if I can use Magick.net to upsample or upscale (not sure of the correct term) the low res images to create a higher resolution and dpi for printing of those same images.

**Comment by dlemstra** [comment]
I am not sure what you are asking. There is no "CSI trick" to make a low res image a high res image. But changing the DPI (density) might work for your situation. It would be better to use another tool that just changes the density of an image without decoding and encoding the image. Exiftool could be used for that.
